db/seeds.rb

```diff
@@ -51,7 +51,7 @@
   },
   {
     application: "LTI Starter App",
-    tenant: "starter-app",
+    tenant: Rails.application.secrets.default_lti_key,
```

Comment:
ok, I guess that is a good change. I just don't think it needs to be in the secrets. IMO